docs/docs/features/environments.mdx

```diff
@@ -30,8 +36,24 @@ the framework you're using, some environment variables are not exposed by defaul
 appropriate prefix, e.g. `NEXT_GROWTHBOOK_CLIENT_KEY='sdk-abc123'` in order to access environment variables in Next.js
 client code.
 
+## Environment Inheritance
+
+When creating an environment via the API, you can specify a `parent` environment it should inherit from.
```

Comment:
Possible edit:
When you create an environment, you can specify a parent environment to inherit from. The new environment initially copies all feature rules from its parent. After creation, the environments operate independently—changes to feature rules in one environment won't affect the other. Think of it like creating a fork or snapshot of the parent environment.